tests/foreman/cli/test_repository.py

```diff
@@ -139,6 +142,24 @@ def gpg_key(module_org):
     return make_content_credential({'organization-id': module_org.id})
 
 
+@pytest.fixture(scope='class')
+def host_setup(module_org):
```

Comment:
I think could be useful at the global level. the scoping is a bit more containing since most tests don't fall under a class, but it could be more generalized.
```suggestion
def host_setup(module_org, class_target_sat):
```